test/functional/abc-magnetic-anomaly-mining.py

```diff
@@ -82,8 +82,8 @@ def update_time():
             for _ in range(len(inputs)):
                 addr = mining_node.getnewaddress()
                 output = {
-                    # 50 BCH per coinbase
-                    addr: decimal.Decimal(50)
+                    # 26 Logos per coinbase
+                    addr: decimal.Decimal(260)
```

Comment:
Joy. Is this a special bitcoin decimal?

Reply:
It’s the good old 100’000’000.

We should move to 10^9 at some point, but not in this PR.